### PR TITLE
Issues reported on transifex.

### PR DESCRIPTION
**Pull request by jbialobr**
https://www.transifex.com/git-extensions/git-extensions/translate/#nl/ui-master/129793268
https://www.transifex.com/git-extensions/git-extensions/translate/#de/plugins-master/129793089
https://www.transifex.com/git-extensions/git-extensions/translate/#de/plugins-master/129793096

### CODE REVIEW

**Comment by jbialobr** [comment]
@RussKie I added another one 396f63f73a1f45a0729337a7d6115b0ddaafa868 could you have a look at it?

File: GitUI/CommandsDialogs/FormDeleteRemoteBranch.resx [hunk 1]
```diff
@@ -118,7 +118,10 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="labelDeleteBranchWarning.Text" xml:space="preserve">
-    <value>When you delete a branch the commits can get lost because nothing points to them.
-Make sure that commits under remote branch(-es) to be deleted are reachable before proceeding.</value>
+    <value>The commits referenced by the deleted branch will be lost
+if no other branch or tag references them. Make sure there
+is a branch or tag referencing the commits you don't want to loose.
+If you accidentaly deleted a wrong branch, go to the Commands | Show reflog form
+in order to recover the lost commits.</value>
```
Comment (RussKie) [inline GitUI/CommandsDialogs/FormDeleteRemoteBranch.resx]: Few typos.
Here's my corrected version:

```
The commits referenced by the deleted branch will be lost
if no other branch or tag reference them. Make sure there
is a branch or tag referencing the commits you wish to keep.
If you have accidentally deleted a wrong branch, go to the Commands | Show reflog form
in order to recover the lost commits.
```

I'm split on `no other branch or tag references them.` I think it should say `reference`...
@vbjay are you a native speaker to tell the right one?

Comment (RussKie) [reply GitUI/CommandsDialogs/FormDeleteRemoteBranch.resx]: One of my native speaking colleagues reckons it has to be "references", so let's go with that.

```
The commits referenced by the deleted branch will be lost
if no other branch or tag references them. Make sure there
is a branch or tag referencing the commits you wish to keep.
If you have accidentally deleted the wrong branch, go to Commands | Show reflog form
in order to recover the lost commits.
```

Comment (jbialobr) [reply GitUI/CommandsDialogs/FormDeleteRemoteBranch.resx]: Why have you also changed 
`deleted a wrong branch` to `deleted the wrong branch`? It is contrary to my understanding of the language.

**Comment by RussKie** [comment]
Native speaker's correction too

**Comment by RussKie** [comment]
THE native's speaker, lol

**Comment by RussKie** [comment]
My grammar is absent today...

**Comment by jbialobr** [comment]
I corrected the warning text.

**Comment by RussKie** [comment]
Merge away, I'll regenerate and publish the update

**Pull request merged**